[IMP] Inventory: Updating LIFO removal strategy (#15613)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/lifo.rst

Reviewer: Optional: If it's just the app I usually just bold it then use the menuselection rst

```suggestion
for seven cinder blocks by navigating to the **Sales app** and creating a new
```

transfer is created for the lot/serial number that has most recently entered the stock (the **last**
lot/serial number that entered the warehouse's inventory).

.. warning::
   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy is banned, since it can
   potentially result in old, expired, or obsolete products being delivered to customers.

.. _inventory/warehouses_storage/lifo_enable:

Enabling the LIFO removal strategy
==================================

Lots and serial numbers differentiate identical products and track information like arrival or
expiration dates. To enable this feature, navigate to :menuselection:`Inventory --> Configuration
--> Settings`. Under the :guilabel:`Traceability` section, check the box beside :guilabel:`Lots &
Serial Numbers` to enable the feature.

.. image:: lifo/enable-lots-serial-numbers.png
   :alt: Enable Lots & Serial Numbers in the Inventory settings.

Enable tracking by lots or serial numbers
-----------------------------------------

Next, ensure the intended product is tracked by lots or serial numbers by navigating to the product

> form through :menuselection:`Inventory --> Products --> Products` and selecting the desired product.

Reviewer: ```suggestion
form in the :menuselection:`Inventory app --> Products --> Products` , then selecting the desired product.
```

Reviewer: The style convention is to include 'app' in the menuselection code.

> On the product form, open the :guilabel:`General Information` tab and select the check box next to

Reviewer: ```suggestion
On the product form, open the :guilabel:`General Information` tab, select the checkbox next to
```

:guilabel:`By Lots`.

.. image:: lifo/track-by-serial-number.png
   :alt: Enable tracking by serial number.

Assigning lots and serial numbers
---------------------------------

After enabling the features, assign lot or serial numbers to products using an inventory adjustment
or during reception.

By inventory adjustment
~~~~~~~~~~~~~~~~~~~~~~~

Open the :guilabel:`Physical Inventory` page by clicking :menuselection:`Inventory app -->
Operations --> Adjustments --> Physical Inventory`.

To change an existing lot or serial number, click the lot or serial number for the product. On the
:guilabel:`Lot/Serial Number` form, adjust the lot or serial number.

To create a new lot or serial number, on the :guilabel:`Physical Inventory` page, click
:guilabel:`New`. Specify the :guilabel:`Product` and :guilabel:`Lot/Serial Number`. Specify the
:guilabel:`Counted Quantity`. Click :guilabel:`Save`. At this stage, the count (inventory
adjustment) is recorded, but not yet applied. This means that the quantity on hand before the
adjustment has not yet been updated to match the new, real counted quantity. Click the

> :guilabel:`Apply` button at the far right side of the page.

Reviewer: ```suggestion
To create a new lot or serial number on the :guilabel:`Physical Inventory` page, click
:guilabel:`New`. Enter the :guilabel:`Product`, :guilabel:`Lot/Serial Number`, and
:guilabel:`Counted Quantity`, then click :guilabel:`Save`. 

The count (inventory adjustment) is recorded but not yet applied, so the quantity on hand remains 
unchanged. To update it, click the :guilabel:`Apply` button at the far right.
```

Reviewer: Hi Erin, I revised this paragraph to flow better. Feel free to edit or change the suggestion.

   :guilabel:`Quantity` column matches the :guilabel:`Demand` at the top.

Setting removal strategies
--------------------------

After lot or serial numbers have been assigned, set the removal strategy on the product category or
storage location.

.. image:: lifo/location-categories.png
   :alt: Find the Locations or Categories from the Configuration menu.

On the location
~~~~~~~~~~~~~~~

Open :menuselection:`Inventory app --> Configuration --> Locations`. Select the desired location. On
the location form, under the :guilabel:`Logistics` heading, select :guilabel:`Last In First Out
(LIFO)` from the list of removal strategies.

.. image:: lifo/lifo-on-location.png
   :alt: Set Last In First Out as the location removal strategy.

On the product category
~~~~~~~~~~~~~~~~~~~~~~~

Configure removal strategies on product categories by going to :menuselection:`Inventory app -->
Configuration --> Categories` and selecting the intended product category. Next, in the
:guilabel:`Force Removal Strategy` field, specify :guilabel:`Last In First Out (LIFO)`.

.. image:: lifo/lifo-on-category.png
   :alt: Set Last In First Out as the product category removal strategy.

.. important::

>    When there are different removal strategies applied on both the location and product category for

Reviewer: ```suggestion
   When there are different removal strategies applied to both the :guilabel: `Location` and :guilabel:`Product Category` for
```

Is 'location' and 'product category' field references? If so, I recommend adding a GUI label.

   a product, the value set on the :guilabel:`Force Removal Strategy` field set on a product
   category form is applied as top priority.

Workflow
========

> Consider the following example, with the product, `Cinder Block`, which is tracked :guilabel:`By

Reviewer: This whole section can be used in an Example callout; you could even remove the workflow header.

expiration dates. To enable this feature, navigate to :menuselection:`Inventory --> Configuration
--> Settings`. Under the :guilabel:`Traceability` section, check the box beside :guilabel:`Lots &
Serial Numbers` to enable the feature.

.. image:: lifo/enable-lots-serial-numbers.png
   :alt: Enable Lots & Serial Numbers in the Inventory settings.

Enable tracking by lots or serial numbers
-----------------------------------------

Next, ensure the intended product is tracked by lots or serial numbers by navigating to the product
form in the :menuselection:`Inventory app --> Products --> Products` , then selecting the desired
product. On the product form, open the :guilabel:`General Information` tab, select the checkbox next
to the :guilabel:`Track Inventory` field, then choose either :guilabel:`By Unique Serial Number` or
:guilabel:`By Lots`.

.. image:: lifo/track-by-serial-number.png
   :alt: Enable tracking by serial number.

Assigning lots and serial numbers
---------------------------------

After enabling the features, assign lot or serial numbers to products using an :doc:`inventory
adjustment <../../warehouses_storage/inventory_management/count_products>` or :ref:`during

> reception <inventory/product_management/assign-lots>`.

Reviewer 2: ```suggestion
adjustment <../../warehouses_storage/inventory_management/count_products>` or :ref:`during reception 
<inventory/product_management/assign-lots>`.
```

early line break

quotation.

:guilabel:`Confirm` the sales order to create a delivery order. Doing so reserves the newest lot
numbers using the :abbr:`LIFO (Last In, First Out)` removal strategy.

> To view the detailed pickings, click the :guilabel:`Details` link, located on the far-right of the

Reviewer: ```suggestion
To view the detailed pickings, click the :guilabel:`Details` link at the far-right of the
```
